build/mediaelement-and-player.js

```diff
@@ -3266,7 +3266,7 @@ if (typeof jQuery != 'undefined') {
 					for (var j = 0, jl = keyAction.keys.length; j < jl; j++) {
 						if (e.keyCode == keyAction.keys[j]) {
 							if (typeof(e.preventDefault) == "function") e.preventDefault();
-							keyAction.action(player, media, e.keyCode);
+							keyAction.action(player, media, e);
```

Comment:
I wonder if it would be better to add `e` as a 4th parameter, so that `e.keyCode` remains in 3rd place and therefore retains backward compatibility?